tracing/src/test/java/org/axonframework/extensions/tracing/OpenTraceHandlerInterceptorTest.java

```diff
@@ -39,16 +39,20 @@ public void after() {
         mockTracer.scopeManager().active().close();
     }
 
+    static class MyEvent {
```

Comment:
Nit: would you mind making this class `private`?

Reply:
check